### ISSUE TITLE
CocoaPods: No such module 'Usercentrics' [BUG]

### ISSUE THREAD

**Issue by m-schmale** [Usercentrics/flutter-sdk#88]
Hi,

**Describe the bug**
If we use the latest version of CocoaPods, we get the following error:

> Failed to build iOS app
Swift Compiler Error (Xcode): No such module 'Usercentrics'
/Users/ci/.pub-cache/hosted/pub.dev/usercentrics_sdk-2.10.0/ios/Classes/Bridge/AcceptAllBridge.swift:0:7

The error occurs with version 1.15.1 of CocoaPods. It still works with version 1.14.2.

Could you please assist us with this?

We were able to reset CocoaPods to the still working version with these commands if anyone else encounters this issue:

```
brew uninstall cocoapods
wget https://raw.githubusercontent.com/Homebrew/homebrew-core/e8e04dd35a5151d5fa1f094aacf404277f552c40/Formula/c/cocoapods.rb
brew install -s cocoapods.rb
```

**To Reproduce**
1. Build app with CocoaPods 1.14.2
1. The build process is successful
2. Update CocoaPods to 1.15.1
3. Now, the build fails with the previously mentioned error message

**Expected behavior**
The build shouldn't fail obviously.

**Flutter doctor**
>Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 3.13.1, on macOS 14.1.2 23B92 darwin-arm64, locale de-DE)
[✓] Android toolchain - develop for Android devices (Android SDK version 34.0.0)
[✓] Xcode - develop for iOS and macOS (Xcode 15.0.1)
[✓] Chrome - develop for the web
[✓] Android Studio (version 2023.1)
[✓] VS Code (version 1.86.0)
[✓] VS Code (version 1.73.1)
[✓] Connected device (3 available)
[✓] Network resources
>
> • No issues found!

**usercentrics-flutter-sdk branch/commit**
Usercentrics 2.11.3

**Comment by Merlinski** [comment]
Hi,
same problem here.

**Comment by uc-rreis** [comment]
Hey guys, 

I'll have a look and let you know as soon as I have any updates.

Cheers,
Rui

**Comment by uc-rreis** [comment]
Also, what happens if you try to install cocoapods using gem instead of brew, like
```sudo gem install cocoapods```.
This is actually a better way to manage your cocoapods versions, as you can simply do
```sudo gem uninstall cocoapods``` to rollback a cocoapods upgrade to an older version.
I have tried the following:
1.  Install cocoapods 1.14.2, 
2. `pod install`
3.  build/run app
4. upgrade cocoapods to 1.15.2
5. pod install
6. run app
This worked successfully, always managed to run app (in Xcode).

Can you please try that?

Cheers,
Rui

**Comment by m-schmale** [comment]
It seems to be fixed in 1.15.2. Thanks for your quick response. 👍
